mitmproxy/addons/export.py

```diff
@@ -106,6 +107,69 @@ def httpie_command(f: flow.Flow) -> str:
     return cmd
 
 
+def python_requests_command(f: flow.Flow) -> str:
+    # Copy as Python Requests
+    request = cleanup_request(f)
+    request = pop_headers(request)
+
+    def escape_quotes(s: str) -> str:
+        return s.replace('"', '\\"')
+
+    code = "import requests\n\n"
+
+    code += f'url = "{request.pretty_url}"\n'
+    code += "headers = {\n"
+    cookies = ""
+    for k, v in request.headers.items(multi=True):
+        if k.lower() == "cookie":
+            cookies += f"{v};"
+        else:
+            code += f'    "{k}": "{escape_quotes(v)}",\n'
```

Comment:
This will generate invalid code for repeating headers - I'd say it would be ok to go with multi=False here and then manually remove the cookie header. Maybe something like this works with some tweaking? (untested)
```python
request.headers.pop("cookie", None)
header_string = pprint.pformat(dict(request.headers.items()))
```